terra-jupyter-python/Dockerfile

```diff
@@ -91,6 +91,8 @@ ENV USER jupyter
 USER $USER
 # We want pip to install into the user's dir when the notebook is running.
 ENV PIP_USER=true
+# Enable Intel oneDNN optimizatoin by default
+ENV TF_ENABLE_ONEDNN_OPTS=1
```

Comment:
does it make sense to have this in the intel image instead?

Reply:
Hi 
Since TensorFlow v2.5, oneDNN optimization is in the release by default, so users could easily leverage the oneDNN optimization by changing an environment variable.
You could refer to below release notes from official TensorFlow v2.5 release.
https://github.com/tensorflow/tensorflow/releases/tag/v2.5.0
![image](https://user-images.githubusercontent.com/21761437/141349091-395eba7d-2d80-443a-a8ed-3811773db0f9.png)

It doesn't require additional software packages or source codes changes, so it doesn't need to be in the specific image like Intel image.

thanks
Louie